### DISCUSSION TITLE
Trajectories library for different video motions?

### DISCUSSION THREAD

**Discussion by Jonseed** [thygate/stable-diffusion-webui-depthmap-script#164] [Q&A]
Has anyone made a library of different trajectories that can be used to adjust the video motion? The default trajectories are ok, but I'd like to try some others. Has anyone compiled different options anywhere? Do you just put these numbers into the translate and crop fields?

**Comment by thygate** [comment]
these are the params for the four demo video's:

trajectories : ['double-straight-line', 'double-straight-line', 'circle', 'circle'], 
crop border : [0.03, 0.03, 0.05, 0.03], 
x_shift_range : [0.00, 0.00, -0.015, -0.015], 
y_shift_range : [0.00, 0.00, -0.015, -0.00], 
z_shift_range : [-0.05, -0.05, -0.05, -0.05],

**Reply by Duemellon** [reply]
Is there a way to edit these? or add additional actions? 
And is there a way to plot a sequential movement set?

The way I'm reading the tool, even if it goes beyond what the mesh created it inpaints to solve the missing parts, correct?
